app/tabular_metadata.rb

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env ruby
+require 'active_record'
+require 'active_support/core_ext/hash'
+require 'yaml'
+require 'nokogiri'
+require 'rack'
+require 'pry-byebug'
+
+module TabularMetadata
+  class API
+    @settings = YAML.load_file('config/settings.yml')
+
+    ActiveRecord::Base.establish_connection(
```

Comment:
This establish_connection process seems to appear in many files. Can this be split out into a helper/util function?